src/js/containers/generateFiles/GenerateFilesContainer.jsx

```diff
@@ -60,15 +62,30 @@ class GenerateFilesContainer extends React.Component {
 			},
 			d1Status: "waiting",
 			d2Status: "waiting",
-			errorDetails: ""
+			errorDetails: "",
+			gtas: null
 		};
 	}
 
 	componentDidMount() {
 		this.isUnmounted = false;
 		this.checkForPreviousFiles();
+		this.isGtas();
 	}
 
+	isGtas() {
+		ReviewHelper.isGtasWindow()
+			.then((res) => {
+				if(res.open != this.state.gtas) {
+					this.setState({gtas: res})
```

Comment:
Since all we're passing back now is the "data" element in the JSON (which is either `null` or an object) it might make sense to set gtas to `res.data`)